.github/workflows/ci.yaml

```diff
@@ -0,0 +1,51 @@
+name: test
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "main"
+      - "ci-test"
```

Comment:
Put it there to have a branch for any ci testing.

Reply:
that's a neat idea - so if we change this file in the future, we should use this same branch to do it?

Reply:
Sorry, what's the case where we would want to merge into "ci-test" instead of "main"?

Reply:
this is only to ensure the CI change got triggered on this branch to verify it is correct. If we don't want it there, I can do a follow up to remove it.

Reply:
Sorry @juliev0  I confused you! :D I misused this setting which is meant for identifying branch `targeting` to not currently on. Thanks for pointing out.